controllers/process.js

```diff
@@ -50,4 +47,57 @@ module.exports = ((req, res) => {
       ua.event('Entries', 'New entry error').send()
     }
   })
-})
+}
+
+function verifyCAPTCHA(req, res) {
+  return new Promise((resolve, reject) => {
+    staticman
+      .getSiteConfig()
+      .then(siteConfig => {
+        if (!siteConfig.get('reCaptcha.enabled')) {
+          return resolve()
+        }
+
+        if(
+          req.body.options.reCAPTCHA === undefined ||
+          req.body.options.reCAPTCHA.siteKey === undefined ||
+          req.body.options.reCAPTCHA.encryptedSecret === undefined
```

Comment:
Could we replace this with:

```js
if (!req.body.options.reCAPTCHA || !req.body.options.reCAPTCHA.siteKey || !req.body.options.reCAPTCHA.encryptedSecret) {}
```

It makes the expression nicer to the eye and also checks against other falsy values.